examples/with-mobx/components/Clock.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { inject, observer } from 'mobx-react'
+
+const Clock = inject('store')(observer(({store}) => {
```

Comment:
@fdidron Yep. I get that. But I like to have UI components separated.
Anyway, here this is not a big deal.
We just wanna show how to use Mobx with Next.js.